### PR TITLE
populate shedyamls for all data managers; add two missing

### PR DESCRIPTION
**Pull request by martenson**

### CODE REVIEW

File: data_managers/data_manager_gatk_picard_index_builder/.shed.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/data_manager_gatk_picard_index_builder
-owner: devteam
+categories:
+- Data Managers
+description: data_manager_gatk_picard_index_builder
+long_description: |
+    data_manager_gatk_picard_index_builder
```
Comment (nsoranzo) [inline data_managers/data_manager_gatk_picard_index_builder/.shed.yml]: `description` needs some love, `long_description` can be empty in this case.

File: data_managers/data_manager_picard_index_builder/.shed.yml [hunk 2]
```diff
@@ -1,4 +1,10 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/data_manager_picard_index_builder
-owner: devteam
+categories:
+- Data Managers
+description: data_manager_picard_index_builder
```
Comment (nsoranzo) [inline data_managers/data_manager_picard_index_builder/.shed.yml]: `description` needs some love here too.

File: data_managers/data_manager_picard_index_builder/.shed.yml [hunk 3]
```diff
@@ -1,4 +1,10 @@
...
+homepage_url: ''
+long_description: |
+    data_manager_picard_index_builder
```
Comment (nsoranzo) [inline data_managers/data_manager_picard_index_builder/.shed.yml]: Better empty.

**Comment by nsoranzo** [comment]
Not sure it is important: many `.shed.yml` don't have `homepage_url:`, others have it empty.

**Comment by martenson** [comment]
@nsoranzo thanks, also you have push rights to my fork, feel free to directly edit

**Pull request merged**